cost/budget_report_alerts/budget_report_alerts.pt

```diff
@@ -59,13 +68,21 @@ parameter "param_email" do
   description "A list of email addresses to notify"
 end
 
+###############################################################################
+# Authentication
+###############################################################################
+
 credentials "auth_flexera" do
   schemes "oauth2"
   label "flexera"
   description "Select Flexera One OAuth2 credentials"
   tags "provider=flexera"
 end
 
+###############################################################################
```

Comment:
Datasources and scripts should now be in the same section. You might see them separated in older policies but that will change as we update them.

See: https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Basic-Policy-Structure